ui/src/app/utils/constants.tsx

```diff
@@ -87,6 +87,3 @@ export const STATE_CODE_MAPPING = {
 export const JUPYTER_FILE_EXT = '.ipynb';
 export const RMD_FILE_EXT = '.Rmd';
 export const R_FILE_EXT = '.R';
-
-// name of the tab for accessing notebooks and runtimes, also used to construct URLs
-export const NOTEBOOKS_TAB_NAME = 'notebooks';
```

Comment:
moved to user-apps-utils.

There was a problem when it was located here (which I did not understand well) related to routing.